client/src/Login.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import Web3 from "web3";
+
+let web3: Web3 | undefined = undefined; // Will hold the web3 instance
+
+const Login = ({ onLoggedIn }: any) => {
```

Comment:
@Jaxonwht any type? maybe need to use interface or sth

Reply:
Create a `LoginProps`.

Reply:
interface LoginProps {
  onLoggedIn: () -> void;
}